src/server/pps/server/api_server.go

```diff
@@ -471,21 +461,23 @@ func (a *apiServer) authorizePipelineOpInTransaction(txnCtx *txnenv.TransactionC
 	// (if it did exist, we'd have to check that the user has permission to write
 	// to it, and this is simpler)
 	if output != "" {
-		var required auth.Scope
+		var required auth.Permission
 		switch operation {
 		case pipelineOpCreate:
 			if _, err := txnCtx.Pfs().InspectRepoInTransaction(txnCtx, &pfs.InspectRepoRequest{
 				Repo: &pfs.Repo{Name: output},
 			}); err == nil {
 				// the repo already exists, so we need the same permissions as update
-				required = auth.Scope_WRITER
-			} else if !isNotFoundErr(err) {
+				required = auth.Permission_REPO_WRITE
+			} else if isNotFoundErr(err) {
+				return nil
+			} else {
 				return err
 			}
 		case pipelineOpListDatum, pipelineOpGetLogs:
-			required = auth.Scope_READER
+			required = auth.Permission_REPO_READ
```

Comment:
`authorizePipelineOp` is something I would love to eliminate but it's a little outside the scope of this PR.